### DISCUSSION TITLE
Transfer asdf-yarn

### DISCUSSION THREAD

**Discussion by kroleg** [mise-plugins/.github#9] [Transfer Request]
I am thinking about migrating my org from `adsf` to `mise`, so i want to "secure" asdf-yarn in mise-plugins. 

There is an [issue](https://github.com/twuni/asdf-yarn/issues/37) in [twuni/asdf-yarn](https://github.com/twuni/asdf-yarn) repo ([the "official" one](https://github.com/asdf-vm/asdf-plugins/blob/master/plugins/yarn)) to transfer it to asdf-community. Not sure how much time will it take to transfer though.

We can't transfer `twuni/asdf-yarn` repo both to asdf-community and to mise-plugins. Also if we have asdf-yarn in both they might diverge at some point (which is probably ok?). Would it make sense to create a fork of `twuni/asdf-yarn` and transfer it to mise-plugins?

**Comment by jdx** [comment]
I'm open to whatever you'd like. Plugins labeled "asdf-" will always retain compatibility with asdf though.

**Reply by kroleg** [reply]
Now when you mentioned it, what's difference between `mise-` and `asdf-` plugins? Like i see that [mise-jless](https://github.com/mise-plugins/mise-jless/tree/main/bin) is almost identical to `asdf-` plugins. Also difference is not explained on https://github.com/mise-plugins. Would it make sense to rename this plugin `mise-yarn`? 

Let's proceed with moving fork of `twuni/asdf-yarn` to mice-plugins. (Meahwhile i am working on transferring `twuni/asdf-yarn` to `asdf-community` But again it might take a while).

**Reply by jdx** [reply]
I renamed some initially with the idea I would rename all of them but it's actually a fair amount of work so I decided to just leave the asdf- prefix for now.

Go ahead and transfer it to me and I'll put it into the org.

**Reply by kroleg** [reply]
just an fyi in case you missed it: i started a transfer of my fork (https://github.com/kroleg/asdf-yarn) to you

**Reply by jdx** [reply]
apologies, it looks like I missed it and the transfer expired. Mind trying it again?

**Reply by kroleg** [reply]
no worries. Initiated again

**Comment by kroleg** [comment]
Done
https://github.com/mise-plugins/asdf-yarn
https://github.com/mise-plugins/registry/blob/5ffbcc134f27800109bb65335b4b9423742b6807/plugins/yarn
